build/build.c

```diff
@@ -172,6 +172,7 @@ rpmRC doScript(rpmSpec spec, rpmBuildFlags what, const char *name,
     return rc;
 }
 
+
```

Comment:
Stray newline added.

Like said in an earlier comment, the addition of rpmtsFromPyObject() could be merged with this commit because it doesn't make much sense alone, especially without any rationale in the commit message. And here too: explain why this is added in the message.